src/test/junitParser.test.ts

```diff
@@ -0,0 +1,29 @@
+import { test } from 'node:test';
+import { parseTestReports } from '../junitParser';
+
+const DIRNAME = __dirname;
```

Comment:
Why not `import.meta.dirname`?

Reply:
The 'import.meta' meta-property is not allowed in files which will build into CommonJS output.
this is the error I get once trying to use `import.meta.dirname`

Reply:
@atlowChemi

Reply:
Oh so let's transpile to ESM. 
Coulsd you add `"type": "module"` to the `package.json`?

Reply:
@atlowChemi Tried again to find solution but couldn't find one its annoying

Reply:
@ETtestim  try now